bot.py

```diff
@@ -4,14 +4,29 @@
 from ollama import Client, ChatResponse
 
 import os
+import logging
+import sys
+
+class SuppressOutput:
+    def write(self, _):
+        pass
+    def flush(self):
+        pass
+
+if os.getenv("TOKEN"):
+    sys.stderr = SuppressOutput()
+
+logging.getLogger("telegram").setLevel(logging.CRITICAL)
+logging.getLogger("telegram.ext").setLevel(logging.CRITICAL)
+logging.getLogger("httpx").setLevel(logging.CRITICAL)
```

Comment:
I don't think we can rely on the python tg bot not to report the API token in critical levels too. So I would just disable TG logs altogether:

```suggestion
logging.getLogger("telegram").disable()
logging.getLogger("telegram.ext").disable()
logging.getLogger("httpx").disable()
```